app/controllers/oauths_controller.rb

```diff
@@ -8,15 +8,15 @@ def callback
     @user = login_from(provider)
 
     if @user
-      redirect_to root_path, notice: "Вы вошли через #{provider.titleize}"
+      redirect_to root_path, notice: t('flash.ouath_log_in', provider: provider.titleize )
     else
       begin
         @user = create_from(provider)
         reset_session
         auto_login(@user)
-        redirect_to root_path, notice: "Вы вошли через #{provider.titleize}"
+        redirect_to root_path, notice: t('flash.ouath_log_in', provider: provider.titleize )
       rescue
-        redirect_to root_path, alert: "Ошибка входа через #{provider.titleize}!"
+        redirect_to root_path, alert: t('flash.ouath_wrong', provider: provider.titleize )
```

Comment:
Line is too long. [90/80]<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.